.gitignore

```diff
@@ -11,3 +11,10 @@
 /yggd.bash
 /yggdrasil.spec
 /doc/tags.toml
+
+.idea
+
+*.pem
+*.rpm
+*.gz
+*.x86
```

Comment:
Is this a product of cross-compiling? If it is, I think the `/ygg` (and similar) patterns could be updated to handle this kind of build result.

Reply:
I was unable to reproduce creation of files with *.x86 so I will remove it.